CHANGELOG.md

```diff
@@ -0,0 +1,97 @@
+# Changelog
+
+## [Unreleased]
+
+## [v0.1.5] - 2021-01-05
+
+### Added
+
+- Allow support for semicolumn and tabulation in CSV files
+- Collect current distance from OR-Tools
+- Allow initial routes when clustering methods are required
+- Position field for activities
+- Sheduling and initial routes are now compatible
+- Unfeasible missions are removed from routes [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- Log reasons of unfeasible missions [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Read and log OR-Tools version
+- Route costs [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+- Log of matrix computation [#52](https://github.com/Mapotempo/optimizer-api/pull/52)
+- Allow multidepot within clustering [#56](https://github.com/Mapotempo/optimizer-api/pull/56)
```

Comment:
We should mention that this is a preliminary version and not production-ready and for testing and it will continue to evolve

Reply:
I'll add a WIP mention